.github/workflows/dev_compile.yml

```diff
@@ -0,0 +1,233 @@
+---
+name: Compile Develop Brench
```

Comment:
@bmorcelli  - Just checking, if we need this dev_compile.yml file?

Reply:
No, we don´t need... delete it please

I'm sorry, it just slipped out..